x11/xorg-xorgproto/Portfile

```diff
@@ -3,13 +3,12 @@
 PortSystem       1.0
 
 name             xorg-xorgproto
-epoch            1
```

Comment:
A port's epoch can never be removed or decreased. See https://guide.macports.org/#reference.keywords